soc/arm/nxp_imx/rt/Kconfig.defconfig.series

```diff
@@ -32,6 +32,10 @@ config GPIO_MCUX_IGPIO
 	default y if HAS_MCUX_IGPIO
 	depends on GPIO
 
+config PINCTRL_IMX
+	default y if HAS_MCUX_IOMUXC
+	depends on PINCTRL
```

Comment:
see my previous comment

Reply:
Explained in the comment above, basically want to avoid having a bunch of DT compat strings in the KConfig